src/options.rs

```diff
@@ -14,27 +14,31 @@ impl Default for SetterPattern {
     }
 }
 
-#[derive(Debug)]
+#[derive(Debug, Clone)]
 pub struct Options {
     /// e.g. `#[setters]` (defaults to true)
     setter_enabled: bool,
     /// e.g. `#[setters(owned)]` (defaults to mutable)
     setter_pattern: SetterPattern,
     /// e.g. `#[setters(prefix="with")]` (defaults to None)
-    setter_prefix: Option<String>,
+    setter_prefix: String,
     /// e.g. `#[setters(private)]` (defaults to public)
     setter_public: bool,
     /// e.g. `#[setters(options="implicit")]` (defaults to explicit)
     setter_implicit_options: bool,
     /// e.g. `#[getters]` (defaults to false)
     getter_enabled: bool,
     /// e.g. `#[getters(prefix="with")]` (defaults to None)
```

Comment:
I know this is not yet implemented, but to avoid confusion, I'd not use a different prefix example for getters, e.g. `get`. Also, I'd mention that this get's concatenated with the field name like `prefix_fieldname`.